hbshelpers/assign.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function assign(...options) {
```

Comment:
when I talked with Tom about this 2 weeks ago he didn't really like the name "assign". Maybe something like "shallowMergeConfig"? 3 words is a little extra, but I think we want some way to signify that this only does a shallow merge as opposed to a deep merge, and that this is only for static config.